doc/usecases/record_elektra/UC_change_notifications.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Notify plugins when data in the KDB changes
+
+## Summary
+
+- **Title:** Notify plugins when data in the KDB changes
+- **Scope:** Configuration/Logging
+- **Level:** Developer goal
+- **Actors:** Plugin developer
+- **Brief:** Some plugins are only executed when something changed in the KDB and as an input receive the keys that changed in their old and new state.
+
+## Scenarios
+
+- **Precondition:**
+  - Something in the KDB changed
+  - A plugin listening to such a hook is enabled
+- **Main success scenario:**
+  - After a change in the KDB the plugin(s) listening on the hooks are notified
+  - The plugins receive both the old and new state of the keys
```

Comment:
Isn't this implementation detail? Actually they want to know the difference? (Duplicate with Brief)